### PR TITLE
Added functionality to utilize a pre-built nGraph built for the bridge

### PR DESCRIPTION
**Pull request by ransford2011**
Added logic to build the nGraph-TF bridge with a pre-built nGraph.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -129,41 +129,42 @@ else()
...
-
-
-if(OS_VERSION STREQUAL "\"centos\"")
-	set(NGRAPH_IMPORTED_LOCATION ${NGRAPH_INSTALL_DIR}/lib64/${LIBNGRAPH_SO})
-else()
-	set(NGRAPH_IMPORTED_LOCATION ${NGRAPH_INSTALL_DIR}/lib/${LIBNGRAPH_SO})
-endif()
 
+if (NOT DEFINED PREBUILT_NGRAPH_INSTALL_PATH)
+    option(NGRAPH_DISTRIBUTED_ENABLE "Add distributed mode to the CPU backend" FALSE)
```
Comment (shssf) [inline CMakeLists.txt]: In case of usage pre-rebuilt nGraph, how enable `NGRAPH_DISTRIBUTED_ENABLE`?

**Comment by ransford2011** [comment]
Thanks for pointing this out @shssf.  I've fixed that issue.

**Comment by avijit-nervana** [comment]
@ransford2011 I am doing a PR to extend this beyond just pre-built `nGraph` - so I will close your PR once mine is ready.

**Comment by avijit-nervana** [comment]
We are closing this as this functionality is available with #316

**Pull request closed without merging**